Emily Xia Assignment 1

emxia18 opened the pull request.

emxia18 committed:
add ch_1_lesson_emily_xia.py to repository

emxia18 committed:
Add files via upload

add ch_1_assign_emily_xia.py

emxia18 committed:
add ch_1_assign_emily_xia.py to repository

emxia18 committed:
Merge branch 'master' of https://github.com/emxia18/software-training

DavidCarlip commented on ch-1_assign_emily_xia.py:
```diff
@@ -0,0 +1,90 @@
+		"Competition_Location":[
+			"Davis_CA"],
+		"Competition_Awards":[
+			"None"]}}
```
You need to ask for user input to retrieve a piece of information about a specific team. Right now, this is just a dictionary that does nothing.

emxia18 replied:
Okay, got it. Thanks! I will try to work on that when I have time. If I have any questions, should I ask them via slack or can I ask them here? Thanks!!

emxia18 replied:
Okay, got it. Thanks! I will try to work on that when I have time. If I have any questions, should I ask them via slack or can I ask them here? Thanks!!

emxia18 committed:
add lines 91-93 on ch_1_assign_emily_xia.py

emxia18 committed:
edit lines 91-101

emxia18 committed:
combine two lists: "Names_of_Competition" and
"Competition_Location" into one dictionary

emxia18 committed:
write the code

emxia18 committed:
add modified version of ch_2_assign_emily_xia.py

emxia18 committed:
add comments and improve code

emxia18 committed:
add search function

emxia18 committed:
add validation in search function

emxia18 committed:
delete string, integer, etc from add part to prevent code from crashing~

emxia18 committed:
remove extra file

emxia18 committed:
add modify function

emxia18 committed:
write code, some validation still needed

emxia18 committed:
add verification, does not take into account first dictionary as part of
the big dictionary

emxia18 committed:
improve validation

emxia18 committed:
write code

emxia18 committed:
remove ; from code

emxia18 committed:
improve code based on comment

emxia18 committed:
improve code based on comments

emxia18 committed:
improve modify function

emxia18 committed:
add validation

add validation for all categories when adding and modifying a team

emxia18 committed:
add validation

change camera_vision_system so that it will only accept "True" or
"False"

emxia18 committed:
add validation

added validation requiring an integer for Rookie_year and boolean for
Competition_Status_2019
